src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -10934,15 +10926,6 @@ private static bool IsNullabilityMismatch(TypeSymbol type1, TypeSymbol type2)
         public override BoundNode? VisitQueryClause(BoundQueryClause node)
         {
             var result = base.VisitQueryClause(node);
```

Comment:
I guess the base method will handle visiting the lambda normally now?

Reply:
Yes, exactly.